plugins/msgrpc.rb

```diff
@@ -67,7 +63,7 @@ def initialize(framework, opts)
 
     # If the run in foreground flag is not specified, then go ahead and fire
     # it off in a worker thread.
-    if (opts['RunInForeground'] != true)
+    unless opts['RunInForeground']
```

Comment:
This changes semantic meaning just like https://github.com/rapid7/metasploit-framework/pull/4823/files#diff-6f4a260f554e69b43241ad5eb01498acR75 . Beware.